src/test/java/mshantadze/SearchTest.java

```diff
@@ -0,0 +1,47 @@
+package mshantadze;
+
+import mshantadze.pages.HomePage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testng.Assert;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+public class SearchTest {
+    private static final Logger LOGGER = LoggerFactory.getLogger(SearchTest.class);
+    private WebDriver driver;
+
+    @BeforeTest
+    public void beforeTests() {
+        this.driver = new ChromeDriver();
+        driver.get("https://www.ebay.com");
+    }
+
+    @AfterTest
+    public void afterTests() {
+        driver.quit();
+    }
+
+    @Test
+    public void searchTest() throws InterruptedException {
```

Comment:
Don't use throws. It's bad if we speak about handling issues. Try catch is much better